ts/DataGrid/DataGrid.ts

```diff
@@ -420,6 +409,52 @@ class DataGrid {
             new DataTable(tableOptions as DataTableOptions);
     }
 
+    /**
+     * Calculates all references to columnIds on all levels below defined level.
+     *
+     * @param columns
+     * Structure that we start calculation
+     *
+     * @param [onlyEnabledColumns=true]
+     * Extract all columns from header or columns filtered by enabled param
+     * @returns
+     */
+    public getColumnIds(
```

Comment:
This doclet needs an update after moving this method to the `DataGrid` class. There is no context from where these `columnIDs` are extracted.